### PR TITLE
fix(ddx): add deprecation warning to esxi docs

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx [hunk 1]
```diff
@@ -54,6 +54,10 @@ If you require SSH access to your Elastic Metal server, you can enable SSH on yo
 
 ## Enabling SSH on ESXi
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx:58]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: dedibox-network/rpn/how-to/configure-rpnv2.mdx [hunk 2]
```diff
@@ -148,6 +148,10 @@ Do not forget to adapt the VLAN ID, Interface Name, and Network configuration to
 
 ## How to configure RPNv2 on ESXi
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline dedibox-network/rpn/how-to/configure-rpnv2.mdx:152]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: dedibox-network/rpn/how-to/use-rpn-san-esxi-v7.mdx [hunk 3]
```diff
@@ -13,6 +13,10 @@ categories:
   - dedibox-network
 ---
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline dedibox-network/rpn/how-to/use-rpn-san-esxi-v7.mdx:17]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: dedibox-network/rpn/how-to/use-rpn-san-esxi.mdx [hunk 4]
```diff
@@ -13,6 +13,10 @@ categories:
   - dedibox-network
 ---
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline dedibox-network/rpn/how-to/use-rpn-san-esxi.mdx:17]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: tutorials/add-disk-configure-datastore-esxi/index.mdx [hunk 5]
```diff
@@ -14,6 +14,11 @@ dates:
   posted: 2020-01-21
 ---
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline tutorials/add-disk-configure-datastore-esxi/index.mdx:18]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: tutorials/configure-ipv6-virtual-machine-esxi/index.mdx [hunk 6]
```diff
@@ -13,6 +13,10 @@ dates:
   posted: 2022-02-24
 ---
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline tutorials/configure-ipv6-virtual-machine-esxi/index.mdx:17]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: tutorials/configure-virtual-machine-esxi/index.mdx [hunk 7]
```diff
@@ -9,10 +9,14 @@ categories:
   - dedibox
 tags: dedibox virtual-machines VMware-ESXi iso-file
 dates:
-  validation: 2025-05-01
+  validation: 2024-07-15
   posted: 2020-01-27
 ---
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline tutorials/configure-virtual-machine-esxi/index.mdx:17]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: tutorials/create-esxi-cluster-dedibox/index.mdx [hunk 8]
```diff
@@ -10,11 +10,13 @@ hero: assets/scaleway_vmware.webp
...
   - dedibox
 dates:
-  validation: 2024-02-08
+  validation: 2024-07-15
   posted: 2020-12-28
 ---
 
-## VMware vSphere - Overview
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```
Comment (nerda-codes) [inline tutorials/create-esxi-cluster-dedibox/index.mdx:18]:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

File: bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx (hunk 1 above)
Comment (ofranc) [reply bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx:58]: Due to VMware's acquisition by Broadcom, the free edition of VMware vSphere Hypervisor is no longer available on the VMware website. For more details, check out [the EOL of free vSphere Hypervisor](https://knowledge.broadcom.com/external/article?legacyId=2107518). Please note that the following content is provided as-is, without any guarantee of functionality.
We suggest considering alternative hypervisors such as Proxmox. You can learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**